collectors/errata/core.py

```diff
@@ -32,24 +32,36 @@ def get(path, return_json=True, session=None, **request_kwargs):
     return response.json() if return_json else response
 
 
-def get_all_errata() -> list[tuple[str, str]]:
+def get_erratum(et_id) -> dict:
+    """
+    get all necessary data for a given erratum
+
+    this is unfortunately one more API query but
+    both list and search miss some timestamps
+    """
+    erratum_json = get(f"/advisory/{et_id}.json")
+    return {
+        "et_id": et_id,
+        "advisory_name": erratum_json["advisory_name"],
+        "created_dt": erratum_json["timestamps"]["created_at"],
+        "updated_dt": erratum_json["timestamps"]["updated_at"],
+    }
+
+
+def get_all_errata() -> list[dict]:
     """
     Fetches IDs for all Errata with CVEs when collector is run for first time
     """
-    all_errata_with_cves = get("/cve/list.json").items()
-    return [
-        (errata_id, errata["advisory"]) for errata_id, errata in all_errata_with_cves
-    ]
+    all_errata_with_cves = get("/cve/list.json").keys()
+    return [get_erratum(errata_id) for errata_id in all_errata_with_cves]
```

Comment:
Could we maybe raise a ticket against ET people to include `updated_at` in the list endpoint? For now we'll have to use this ofc

Reply:
I created a request for them. It is however not just this endpoint but also XMLRPC call we use. We will see how they will be willing to deal with it. If not we will just live with more queries.

Reply:
https://issues.redhat.com/browse/RHELWF-8322